src/build.c

```diff
@@ -91,9 +91,9 @@ typedef struct RunInfo
 static RunInfo *run_info;
 
 #ifdef G_OS_WIN32
-static const gchar RUN_SCRIPT_CMD[] = "geany_run_script.bat";
+static const gchar RUN_SCRIPT_CMD[] = "geany_run_script_XXXXXX.bat";
 #else
-static const gchar RUN_SCRIPT_CMD[] = "./geany_run_script.sh";
+static const gchar RUN_SCRIPT_CMD[] = "geany_run_script_XXXXXX.sh";
 #endif
```

Comment:
This is just a template now, we use full path to refer the file so ./ is not needed.